plugins/generator-1.21.8/neoforge-1.21.8/utils/triggers.java.ftl

```diff
@@ -416,6 +416,23 @@
 </#if>
 </#macro>
 
+<#macro onEntityFallsOn procedure="">
+<#if hasProcedure(data.onEntityFallsOn)>
+@Override public void fallOn(Level world, BlockState state, BlockPos pos, Entity entity, double distance) {
+	super.fallOn(world, state, pos, entity, distance);
+	<@procedureCode data.onEntityFallsOn, {
+		"x": "pos.getX()",
+		"y": "pos.getY()",
+		"z": "pos.getZ()",
+		"world": "world",
+		"entity": "entity",
+		"blockstate": "state",
+		"double": "distance"

```

Comment:
This is not right. Left is dependency name and right is parameter name. Please test the changes you do.

We don't appreciate blind changes, which this is..

Reply:
Do all the changes, test them out manually in game and once sure, make a single commit that describes the change done